### ISSUE TITLE
helmfile status does not apply --namespace option

### ISSUE THREAD

**Issue by mattymo** [roboll/helmfile#1114]
helmfile status checks the default namespace set in the kubectl context, not the one specified in the command line.

Command run:
helmfile --namespace mattymo status
Expected results:
```
Getting status db
NAME: db
LAST DEPLOYED: Wed Feb 19 16:17:56 2020
NAMESPACE: mattymo
STATUS: deployed
REVISION: 1
TEST SUITE: None
...
```
Actual results:
```
err: no releases found that matches specified selector(name=db) and environment(default), in any helmfile
```

**Comment by AlexsJones** [comment]
I've seen something similar  
```
➜  helmfiles git:(master) ✗ helm ls  --all-namespaces
NAME	NAMESPACE  	REVISION	UPDATED                             	STATUS  	CHART     	APP VERSION
gloo	gloo-system	1       	2020-02-21 13:37:10.839352 +0000 UTC	deployed	gloo-1.3.6
➜   helmfile --namespace=gloo-system --file releases/gloo.yaml status
Getting status gloo
in releases/gloo.yaml: release "gloo" failed: helm exited with status 1:
  Error: release: not found
```

And this only e

**Comment by mumoshu** [comment]
@mattymo @AlexsJones Thanks for reporting!

Which versions of helmfile are you using?

I'm asking because this should be fixed since v0.100.1 via #1108

**Comment by mattymo** [comment]
@mumoshu You are the best! It works on v1.100.1.
